app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  validates :name, :email, presence: true
+  has_many :activities
+  has_many :categories
+end
```

Comment:
- [ ] Please validate the input fields for the `Category and transaction`. You have to flash or raise the message if a user tries to create the data without the information or with the empty data.